app/controllers/events_controller.rb

```diff
@@ -1,6 +1,7 @@
 class EventsController < ApplicationController
   before_action :set_event, only: [:show, :edit, :update, :destroy]
-
+  # Uncomment the below to integrate admin auth
+  # before_action :authenticate_admin!, only: [:new, :create, :edit, :update, :destroy]
```

Comment:
probably should use an except statement instead of an only statement but no big deal.

More importantly, add a fake admin to the db in seeds.rb (look up how to seed a db cause I don't remember exactly) so we can use it in our tests or while developing.

Reply:
While this still might be necessary for other reasons, I have testing solutions for both rspec and cucumber for logging in as different user types. Especially rspec as devise allows you to just 'log in' a user created with a factory.